src/main/java/racingcar/controller/Dice.java

```diff
@@ -0,0 +1,14 @@
+package racingcar.controller;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import static racingcar.controller.constants.IntegerConstants.*;
+
+public class Dice {
+    public boolean isCarCanGoForward(){
```

Comment:
```suggestion
    public boolean canCarGoForward(){
```
별 건 아니지만 이렇게 해도 의미는 통할 것 같네요 ㅎㅎ

Reply:
좋은 추천 감사합니다! 저는 canXXX가 되면 boolean에서 true만 반환할 것 같다는 느낌이 들어 거의 이름을 만들때 사용하지 않는데, 해당 메서드는 canXXX이 되어도 잘 읽혔을 것 같다는 생각이 들어요. 감사합니다 👍